.vscode/cspell.json

```diff
@@ -1017,8 +1017,11 @@
         "adla",
         "adls",
         "byoa",
+        "ddls",
         "distcp",
         "eloqua",
+        "ename",
```

Comment:
@annelo-msft this is a quote from service team:

> We have been through multiple times with central api team and got this approved in existing condition.
> We have some constraints that does not allow us to make the code changes.

I'm going to merge this PR, since it is pending for a long time. Let me know if you still have concern.